dist/flags.py

```diff
@@ -117,13 +117,14 @@
         'SESSION_CAN_WAIT',
         'SESSION_INTERNAL',
         'SESSION_LOCKED_CHECKPOINT',
-        'SESSION_LOCKED_HANDLE_LIST',
         'SESSION_LOCKED_METADATA',
         'SESSION_LOCKED_PASS',
+        'SESSION_LOCKED_READ_HANDLE_LIST',
```

Comment:
What would you think of `SESSION_LOCKED_HANDLE_LIST_READ` and `SESSION_LOCKED_HANDLE_LIST_WRITE` so they sort together?